app/client/src/utils/hooks.ts

```diff
@@ -1570,6 +1575,358 @@ function useSharedLayers({
+        symbol: new SimpleMarkerSymbol({
+          style: 'triangle',
+          color: '#93AD34',
+          outline: {
+            style: 'solid',
+            color: '#000',
+            width: 1,
+          },
+        }),
+      }),
+      outFields,
+      popupTemplate: {
+        title: getTitle,
+        content: getTemplate,
+        outFields,
+      },
+    });
+    setLayer('storageTanksLayer', storageTanksLayer);
+    return storageTanksLayer;
+  }
+
+  function getSewerOverflowsLayer() {
+    const outFields = ['facility_name', 'npdes_id', 'dmr_tracking'];
+
+    const sewerOverflowsLayer = new FeatureLayer({
+      id: 'sewerOverflowsLayer',
+      url: services.data.combinedSewerOverflows,
+      title: 'Combined Sewer Overflows',
+      listMode: 'hide-children',
+      visible: false,
+      renderer: new SimpleRenderer({
+        symbol: new SimpleMarkerSymbol({
+          style: 'circle',
+          color: '#833599',
+          size: 4,
+          outline: {
+            style: 'solid',
+            color: '#000',
+            width: 0.7,
+          },
+        }),
+      }),
+      outFields,
+      popupTemplate: {
+        title: getTitle,
+        content: getTemplate,
+        outFields,
+      },
+    });
+    setLayer('sewerOverflowsLayer', sewerOverflowsLayer);
+    return sewerOverflowsLayer;
+  }
+
+  async function getDamsLayer() {
+    const damsLayer = (await Layer.fromPortalItem({
+      portalItem: new PortalItem({
+        id: services.data.dams.portalId,
+      }),
+    })) as __esri.FeatureLayer;
+    damsLayer.id = 'damsLayer';
+    damsLayer.listMode = 'hide-children';
+    damsLayer.title = 'Dams';
+    damsLayer.visible = false;
+    damsLayer.popupTemplate = new PopupTemplate({
+      content: [
+        {
+          type: 'text',
+          text: '<p style="text-align:center;"><span style="font-size:18px;"><strong>{expression/expr0}</strong></span><br><span style="font-size:14px;">({NIDID})</span></p><p><span style="font-size:18px;">Owner Type: {expression/expr60}</span><br><span style="font-size:18px;">Designed for: {expression/expr16}</span><br><span style="font-size:18px;">Year completed: {expression/expr59}</span><br><span style="font-size:18px;">City: {expression/expr5}</span><br><span style="font-size:18px;">State: {STATE}</span><br><span style="font-size:18px;">Condition Assessment: {CONDITION_ASSESSMENT}</span><br><span style="font-size:18px;">Recent Assessment Date: {CONDITION_ASSESS_DATE}</span></p><p style="text-align:center;"><span style="font-size:18px;"><strong>Specifics</strong></span></p><p><span style="font-size:18px;">Type:&nbsp;{expression/expr13}</span><br><span style="font-size:18px;">Core:&nbsp;{expression/expr14}</span><br><span style="font-size:18px;">Foundation: {expression/expr15}</span><br><span style="font-size:18px;">Dam length: {expression/expr38} ft</span><br><span style="font-size:18px;">Dam height: {expression/expr39} ft</span></p>',
```

Comment:
The font size is larger than usual here, too.
```suggestion
          text: '<p style="text-align:center;"><span style="font-size:14px;"><strong>{expression/expr0}</strong></span><br><span style="font-size:14px;">({NIDID})</span></p><p><span style="font-size:14px;">Owner Type: {expression/expr60}</span><br><span style="font-size:14px;">Designed for: {expression/expr16}</span><br><span style="font-size:14px;">Year completed: {expression/expr59}</span><br><span style="font-size:14px;">City: {expression/expr5}</span><br><span style="font-size:14px;">State: {STATE}</span><br><span style="font-size:14px;">Condition Assessment: {CONDITION_ASSESSMENT}</span><br><span style="font-size:14px;">Recent Assessment Date: {CONDITION_ASSESS_DATE}</span></p><p style="text-align:center;"><span style="font-size:14px;"><strong>Specifics</strong></span></p><p><span style="font-size:14px;">Type:&nbsp;{expression/expr13}</span><br><span style="font-size:14px;">Core:&nbsp;{expression/expr14}</span><br><span style="font-size:14px;">Foundation: {expression/expr15}</span><br><span style="font-size:14px;">Dam length: {expression/expr38} ft</span><br><span style="font-size:14px;">Dam height: {expression/expr39} ft</span></p>',
```
Would also be more consistent if the field names are bold.